images/vscode-ssh-server/README.md

```diff
@@ -0,0 +1,94 @@
+# Marinated Concrete's VSCode-compatible ssh server
+This runs a little ssh server that's compatible with VSCode's remote SSH feature. Unfortunately, [the LinuxServer one](https://docs.linuxserver.io/images/docker-openssh-server/) cannot be used because it is based on Alpine so it lacks glibc, a [known limitation](https://code.visualstudio.com/docs/remote/ssh#_remote-ssh-limitations).
+
+It's also less flexible/more opinionated. The intention is to run it as a sidecar to an existing application you have, such as esphome, HomeAssistant, or zwavejs.
+
+## Configuration:
+In a new container alongside your application's container:
+- Set the environment variable `AUTHORIZED_KEYS_URL` to a path containing your authorized keys. For instance, `https://github.com/$GH_USERNAME.keys`.
+- Mount a small empty dir in `/config` - this will container your server's ssh host key, which must be persisted
+- Mount the volume you want accessible over ssh in `/data`. When you ssh in, this is the directory where you'll start.
+
+## Examples
+```yaml
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Why a deployment instead of a `StatefulSet`?

Reply:
1) More people are familiar with them
2) It is a stateless sidecar, so it is suitable for a `Deployment`
3) I really don't think this is a good fit for the main use case of `StatefulSet`s (replicas > 1)
4) I don't think it matters a ton for an example anyways - it's the part of the chrome vs the content

Reply:
It's not stateless when it preserves it's generated ssh key ;)

Reply:
Fair. I was only considering the steady state operations. Though it is stateless if you mount a secret instead!